### PR TITLE
9 create add food component

### LINKED ISSUE #9: Create AddFood component
We're going to need an `AddFood` component that will show a form for the user to create the details for a new food.

>Input fields for 'name', 'CO2 output', and 'water usage'. Drop-down selector for 'food group'/'category'.

```js
{
  "name": "Dragonfruit",
  "category_id":  1,
  "carbon_output": 101,
  "water_usage": 88
}
```

How you render the form visually is up to you - but run your idea past your lead developers before implementing. Here's a hopefully helpful [reference](https://react.semantic-ui.com/collections/form/).

Relates to #12

### PR DESCRIPTION
**Pull request by joanna-klimas**
closes #9

### CODE REVIEW

File: client/components/AddFood.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
```
Comment (bryceknz) [inline client/components/AddFood.jsx]: Unnecessary newline?

File: client/components/AddFood.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (emilyparkes) [inline client/components/AddFood.jsx]: Please also remove the new line in-between import statements.

File: client/components/AddFood.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  handleInputChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  render() {
+    return (
+      <Container>
+        <div className='ui hidden divider'></div>
```
Comment (emilyparkes) [inline client/components/AddFood.jsx]: Please change all html/jsx elements to have `"` double quotes. Thanks!

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: tests/client/components/AddFood.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
```
Comment (emilyparkes) [inline tests/client/components/AddFood.test.js]: Please remove this new line also. New line only needed between external imports and local imports. Thank you! :) 
Tests pass ✅

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: tests/client/components/AddFood.test.js (hunk 4 above)
Comment (joanna-klimas) [reply tests/client/components/AddFood.test.js]: done!

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**